src/pages/privacypolicy.tsx

```diff
@@ -0,0 +1,24 @@
+import Layout from "../hoc/layout";
+import PrivacyPolicys from "../components/ComponentPages/privacyPolicy";
+import { getTermsAndServicesContent } from "src/network/api/termsAndPrivacyApi";
+
+function PrivacyPolicy({ termsAndServicesConrent }) {
+  return (
+    <>
+      <Layout>
+        <PrivacyPolicys termsAndServicesConrent={termsAndServicesConrent} />
+      </Layout>
+    </>
+  );
+}
+export async function getServerSideProps() {
```

Comment:
We need to use `GetStaticProps` instead of `getServerSideProps`. Cause this will not change frequently. We need to generate the page at the build time.

Reply:
Done